homeassistant/components/sensor/worldtidesinfo.py

```diff
@@ -0,0 +1,135 @@
+"""
+Support for retrieving tide levels from worldtides.info.
+
+configuration.yaml
+
+sensor:
+  - platform: worldtidesinfo
+    api_key: "YOUR API KEY"
+    latitude: "LATITUDE OF BEACH"
+    longitude: "LONGITUDE OF BEACH"
+"""
+import logging
+import time
+from datetime import timedelta
+
+import homeassistant.helpers.config_validation as cv
+import requests
+import voluptuous as vol
+from homeassistant.components.sensor import PLATFORM_SCHEMA
+from homeassistant.const import (CONF_API_KEY, CONF_LATITUDE, CONF_LONGITUDE)
+from homeassistant.helpers.entity import Entity
+from homeassistant.util import Throttle
```

Comment:
The [imports](https://www.python.org/dev/peps/pep-0008/#imports) should be ordered.